packages/valaxy-theme-yun/components/YunComment.vue

```diff
@@ -1,14 +1,34 @@
 <script lang="ts" setup>
 import { useRuntimeConfig } from 'valaxy'
+import { ref } from 'vue'
 
+const activeComment = ref<'waline' | 'twikoo'>('waline')
 const runtimeConfig = useRuntimeConfig()
+const isDoubleComment = !!(runtimeConfig.value.addons['valaxy-addon-waline'] && runtimeConfig.value.addons['valaxy-addon-twikoo']) as boolean
+
+function toggleComment() {
+  activeComment.value = activeComment.value === 'waline' ? 'twikoo' : 'waline'
```

Comment:
Known comment systems include waline/twikoo/artalk/gitalk/utterance/giscus/disqus and others.

Although we are not currently developing other comment addons.
I think we should reserve that logic.

Maybe we can use an array and `includes`.

Reply:
How about replace Switch with Select.